src/main/scala/io/github/pauljamescleary/petstore/infrastructure/repository/doobie/DoobieOrderRepositoryInterpreter.scala

```diff
@@ -23,30 +21,40 @@ class DoobieOrderRepositoryInterpreter[F[_]: Monad](val xa: Transactor[F])
       dt => java.sql.Timestamp.from(dt)
     )
 
+  def select(orderId: Long): Query0[orders.Order] = sql"""
+    SELECT PET_ID, SHIP_DATE, STATUS, COMPLETE, ID
+    FROM ORDERS
+    WHERE ID = $orderId
+  """.query[orders.Order]
+
+  def update(order : orders.Order) : Update0 = sql"""
```

Comment:
Is it wrong of me to think that "update" is not a "Query"?

I don't have an analog to go from, not sure if there is a better name than OrderQueries, maybe OrderSQL?

Reply:
Updated it with XxxSQL naming instead, I agree that seems better.